src/app/code/Komoju/Payments/Observer/RestoreQuoteFromSession.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Komoju\Payments\Observer;
+
+use Magento\Checkout\Model\Session as CheckoutSession;
+use Magento\Framework\App\ObjectManager;
+use Magento\Framework\Event\ObserverInterface;
+use Magento\Framework\Event\Observer;
+use Magento\Framework\Session\SessionManager;
+use Magento\Sales\Model\Order;
+use Magento\Quote\Api\CartRepositoryInterface;
+use Psr\Log\LoggerInterface;
+
+class RestoreQuoteFromSession implements ObserverInterface
```

Comment:
@Dinwy We should add some module documentation here explaining why this class exists.

Reply:
Thank you for the feedback! I will add a document for this module. WIP

Reply:
@rramsden I've added [doc block](https://github.com/degica/komoju-magento/pull/14/files#diff-2e6a427ee827e16640c79e37c727b24c2c702bf67c77e536c17fbbfa262f71b6R15) and documentation for [events and observers](https://github.com/degica/komoju-magento/pull/14/files#diff-ab8214b786749cdf3ffa18dcf88f31eccf351ade33354d74f45c322a8fa696bc)
Please check and let me know how you think!